js/console.ts

```diff
@@ -346,14 +346,16 @@ export function stringifyArgs(
       );
     }
   }
-  return out.join(" ");
+  return `${groupIndent}${out.join(" ")}`;
```

Comment:
If the `log` message contains `"\n"`, will be output to the wrong location.

```ts
console.group('g')
console.log("foo\nbar")
console.groupEnd();
```

output:

```
g
  foo
bar
```

Reply:
I think I need to consider how to fix it.

Reply:
@justjavac please review.